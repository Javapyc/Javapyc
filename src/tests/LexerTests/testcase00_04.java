/* This is a comment /*this is still in the first comment // */

class bob { public static void main(String[] ham) {
int x = 4;
int y = 8;
boolean cheese = -this!null;
string Main 
while(((null))) {}
}
