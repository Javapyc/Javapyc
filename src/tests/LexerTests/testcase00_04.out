ReservedWord, class
ID, bob
Delimiter, {
ReservedWord, public
ReservedWord, static
ReservedWord, void
ReservedWord, main
Delimiter, (
ReservedWord, String
Delimiter, [
Delimiter, ]
ID, ham
Delimiter, )
Delimiter, {
ReservedWord, int
ID, x
Delimiter, =
Integer, 4
Delimiter, ;
ReservedWord, int
ID, y
Delimiter, =
Integer, 8
Delimiter, ;
ReservedWord, boolean
ID, cheese
Delimiter, =
Operator, -
ReservedWord, this
Operator, !
ReservedWord, null
Delimiter, ;
ID, string
ID, Main
ReservedWord, while
Delimiter, (
Delimiter, (
Delimiter, (
ReservedWord, null
Delimiter, )
Delimiter, )
Delimiter, )
Delimiter, {
Delimiter, }
Delimiter, }
